netbox/project-static/src/netbox.ts

```diff
@@ -14,6 +14,9 @@ import { initRackElevation } from './racks';
 import { initLinks } from './links';
 import { initHtmx } from './htmx';
 
+// @ts-ignore
+import { cleanGetUrl } from './util';
```

Comment:
The linter will not allow this, as it is unused. Should go away when you implement this properly using event listeners. Remember to remove @ts-ignore